feat(elasticloadbalancingv2): support Weighted Random algorithm and Automatic Target Weights for alb (#30542)
aws/aws-cdk: packages/aws-cdk-lib/aws-elasticloadbalancingv2/README.md

});
```

> ### Weighted random routing algorithms and automatic target weights for Application Loaf balancer
> You can use weighted random routing algorithms by setting `loadBalancingAlgorithmType` to `TargetGroupLoadBalancingAlgorithmType.WEIGHTED_RANDOM`.
> When using this algorithm, Automatic Target Weights (ATW) anomaly mitigation can be used by setting `enableAnomalyMitigation` to true.
> 
> For more information, see [Routing algorithms](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#modify-routing-algorithm) and [Automatic Target Weights (ATW)](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#automatic-target-weights).
> 
> ```ts
> declare const vpc: ec2.Vpc;
> 
> const tg = new elbv2.ApplicationTargetGroup(this, 'TargetGroup', {
>   loadBalancingAlgorithmType: elbv2.TargetGroupLoadBalancingAlgorithmType.WEIGHTED_RANDOM,
>   vpc,
>   enableAnomalyMitigation: true,
> });
> ```

Reviewer: ```suggestion
### Weighted random routing algorithms and automatic target weights for your Application Load Balancer

You can use the `weighted_random` routing algorithms by setting the `loadBalancingAlgorithmType` property.

When using this algorithm, Automatic Target Weights (ATW) anomaly mitigation can be used by setting `enableAnomalyMitigation` to `true`.

For more information, see [Routing algorithms](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#modify-routing-algorithm) and [Automatic Target Weights (ATW)](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#automatic-target-weights).

```ts
declare const vpc: ec2.Vpc;

const tg = new elbv2.ApplicationTargetGroup(this, 'TargetGroup', {
  vpc,
  loadBalancingAlgorithmType: elbv2.TargetGroupLoadBalancingAlgorithmType.WEIGHTED_RANDOM,
  enableAnomalyMitigation: true,
});
```
```

});
```

> ### Weighted random routing algorithms and automatic target weights for your Application Load Balancer

Author: Thanks. I've updated docs.
